pldp/migrations/0009_auto_20190514_1450.py

```diff
@@ -0,0 +1,149 @@
+            model_name='locationline',
+            name='geometry',
+        ),
+        migrations.AddField(
+            model_name='location',
+            name='geometry',
+            field=django.contrib.gis.db.models.fields.GeometryField(default='POLYGON((-101.744384 39.32155, -101.552124 39.330048, -101.403808 39.330048, -101.332397 39.364032, -101.744384 39.32155))', help_text='Polygon or line that describes the geometry of the location. Polygons are intended for counts of people staying in an area while lines are intended forcounts of people moving across a threshold.', srid=4326),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='locationarea',
+            name='location',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='pldp.Location'),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='locationline',
+            name='location',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='pldp.Location'),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='location',
+            name='subdivision',
+            field=models.CharField(blank=True, choices=[('north', 'north'), ('northeast', 'northeast'), ('east', 'east'), ('southeast', 'southeast'), ('south', 'south'), ('southwest', 'southwest'), ('west', 'west'), ('northwest', 'northwest'), ('center', 'center')], help_text='Indication of whether the location is a subdivision of a single survey location.', max_length=9),
+        ),
+        migrations.AlterField(
+            model_name='location',
+            name='type',
+            field=models.CharField(choices=[('line', 'line'), ('area', 'area')], help_text='Indication of whether the location is intended for counts of people moving (across a line), or whether it is intended for counts of people staying (within an area).', max_length=4),
+        ),
+        migrations.AlterField(
+            model_name='locationarea',
+            name='date_measured',
+            field=models.DateField(help_text="Date that the location's attributes were measured."),
+        ),
+        migrations.AlterField(
+            model_name='locationarea',
+            name='total_sqm',
+            field=models.FloatField(blank=True, help_text='Total area of the space defined by the area geometry.', null=True),
+        ),
+        migrations.AlterField(
+            model_name='locationarea',
+            name='typology',
+            field=models.CharField(blank=True, help_text='Typology of the space defined within the area geometry.', max_length=255),
+        ),
+        migrations.AlterField(
+            model_name='locationline',
+            name='date_measured',
+            field=models.DateField(help_text="Date that the location's attributes were measured."),
+        ),
+        migrations.AlterField(
```

Comment:
Looks like the rest of the migrations are maybe some migrations we forgot in a previous PR?

Reply:
Yep I just went through and made sure every help text field had a period at the end for consistency, but figured that could be rolled into a future migration. Sorry I forgot to note that for you!

Reply:
No worries, thanks for the context!